lib/rawHttpRequest/index.js

```diff
@@ -55,6 +55,20 @@ function setupRawHttpRequestOperationConfig (connectorsConfig) {
 						//No change; provide both endpoint and full
 						break;
 				}
+
+				// Add the base URL to the help text for the endpoint input property
+				// so that users know what the base URL actually is
+				const baseUrl = _.get(connectorConfig, 'globalModel.baseUrl');
+				if (_.isString(baseUrl)) {
```

Comment:
It's probably best to also filter out baseUrls that have any mustaching in them (esp if it's `#auth`, which is internal implementation detail we don't want to expose).

Reply:
Ive changed it to remove `#auth.` specifically, as that's the only sensitive part of our implementation.  I think having the dynamic part of the URL as a replaced token is useful, otherwise users may get confused about what the actual URL is and how its generated.. at least having `{{region}}.example.com` gives them a more realistic sense